evaluation/swe_bench/README.md

```diff
@@ -157,6 +157,23 @@ The final results will be saved to `evaluation/evaluation_outputs/outputs/swe_be
 - `report.json`: a JSON file that contains keys like `"resolved_ids"` pointing to instance IDs that are resolved by the agent.
 - `logs/`: a directory of test logs
 
+### Run evaluation with `RemoteRuntime` (experimental)
```

Comment:
The changes in this file seem orthogonal to this PR?

Reply:
oh yeah these are due to the fact that this PR hasn't synced with the latest main - can be solved with a later merge.